satellite/orders/endpoint.go

```diff
@@ -161,6 +181,30 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 			}
 		}
 
+		if err := endpoint.DB.UpdateBucketBandwidthSettle(ctx, bucketID, orderLimit.Action, order.Amount); err != nil {
```

Comment:
When we hit error here, we should "unuse" the serial number and then return the error and not reply with neither accepted nor rejected. That will cause the storage node to retry later.